packages/venia-concept/src/index.css

```diff
@@ -7,6 +7,7 @@ html {
     --venia-border: 224, 224, 224;
     --venia-font: Muli, -apple-system, BlinkMacSystemFont, sans-serif;
     --venia-grey: 246, 246, 246;
+    --venia-orange: 241, 99, 33;
```

Comment:
Look at you, respecting an alphabetized list. 😍

Reply:
I was so excited when I noticed that about this codebase 😆. It's one of those things that always feels nitpicky to bring up but really makes reading and finding things so much easier!